cylc/flow/cfgspec/globalcfg.py

```diff
@@ -283,6 +283,14 @@
                 file size.
             ''')
 
+    with Conf('install'):
+        Conf('source dirs', VDR.V_STRING_LIST, default=['~/cylc-src'], desc='''
+            A list of paths where workflow source directories may be found.
+            This is used to display and install currently non-installed
+            workflows in the UI, and as a search path for installing workflows
+            using ``cylc install REG``.
```

Comment:
Consider
"""A list of paths where ``cylc install`` will look for workflow source directories.
If a workflow source is in one of these directories you can install that workflow with ``cylc install REG`` - rather than giving the full path to the workflow source. Cylc UI uses source folders when searching for non-installed workflows.
"""